packages/xo-web/src/xo-app/sr/tab-disks.js

```diff
@@ -285,7 +309,7 @@ export default class SrDisks extends Component {
     () => this.props.vdis,
     () => this.props.vdiSnapshots,
     () => this.props.unmanagedVdis,
-    concat
+    (vdis, vdiSnapshots, unmanagedVdis) => concat(vdis, vdiSnapshots, sortBy(unmanagedVdis, ['uuid']))
```

Comment:
```suggestion
    (vdis, vdiSnapshots, unmanagedVdis) => concat(vdis, vdiSnapshots, sortBy(unmanagedVdis, 'uuid'))
```

Reply:
Did you choose `uuid` for a reason or just so that it never changes the order?

Reply:
Just to have always the same order.